capa/features/extractors/common.py

```diff
@@ -63,6 +64,13 @@ def extract_format(buf) -> Iterator[Tuple[Feature, Address]]:
         yield Format(FORMAT_FREEZE), NO_ADDRESS
     elif buf.startswith(MATCH_RESULT):
         yield Format(FORMAT_RESULT), NO_ADDRESS
+    elif (
+        buf.replace(b" ", b"").replace(b"\r", b"").replace(b"\n", b"").replace(b"\t", b"").startswith(MATCH_JSON_OBJECT)
```

Comment:
some options:
  - maybe consider only taking a taste of the file, like the first 100 bytes, to avoid all the work of stripping whitespace from parts of the file we won't use here (assume there aren't more than 100 newlines at the start of the file ;-) ).
  - `re.sub("\w", "")` probably replaces all whitespace in one go, rather than the intermediate passes, and is maybe more readable

which would result in something like: `elif re.sub(buf[:100], "\w", "").startswith(MATCH_JSON_OBJECT)`

Reply:
great catch! only doing this on the first 20 bytes should suffice and perform much better than the initial suggestion